### ISSUE TITLE
Summer Resume Raw File

### ISSUE THREAD

**Issue by kylespink13** [kylespink13/Kyle_Spink_Resume#1]
%-------------------------
% Resume in Latex
% Based off of: https://github.com/sb2nov/resume
%------------------------

\documentclass[letterpaper,11pt]{article}
\usepackage{latexsym}
\usepackage{graphicx}
\usepackage[empty]{fullpage}
\usepackage{titlesec}
\usepackage{marvosym}
\usepackage[usenames,dvipsnames]{color}
\usepackage{verbatim}
\usepackage{enumitem}
\usepackage[hidelinks]{hyperref}
\usepackage{fancyhdr}
\usepackage[english]{babel}
\usepackage{tabularx}
\usepackage{hyperref}
% \usepackage[bottom=0in]{geometry}

\input{glyphtounicode}

%----------FONT OPTIONS----------
% sans-serif
% \usepackage[sfdefault]{FiraSans}
% \usepackage[sfdefault]{roboto}
% \usepackage[sfdefault]{noto-sans}
% \usepackage[default]{sourcesanspro}

% serif
% \usepackage{CormorantGaramond}
% \usepackage{charter}

\pagestyle{fancy}
\fancyhf{} % clear all header and footer fields
\fancyfoot{}
\renewcommand{\headrulewidth}{0pt}
\renewcommand{\footrulewidth}{0pt}

% Adjust margins
\addtolength{\oddsidemargin}{-0.5in}
\addtolength{\evensidemargin}{-0.5in}
\addtolength{\textwidth}{1in}
\addtolength{\topmargin}{-.75in}
\addtolength{\textheight}{1.0in}

\urlstyle{same}

\raggedbottom
\raggedright
\setlength{\tabcolsep}{0in}

% Sections formatting
\titleformat{\section}{
  \vspace{-4pt}\scshape\raggedright\large
}{}{0em}{}[\color{black}\titlerule \vspace{-5pt}]

% Ensure that generate pdf is machine readable/ATS parsable
\pdfgentounicode=1

%-------------------------
% Custom commands
\newcommand{\resumeItem}[1]{
  \item\small{
    {#1 \vspace{-2pt}}
  }
}

\newcommand{\resumeSubheading}[4]{
  \vspace{-2pt}\item
    \begin{tabular*}{0.97\textwidth}[t]{l@{\extracolsep{\fill}}r}
      \textbf{#1} & #2 \\
      \small#3 & \small #4 \\
    \end{tabular*}\vspace{-7pt}
}

\newcommand{\resumeSubSubheading}[2]{
    \item
    \begin{tabular*}{0.97\textwidth}{l@{\extracolsep{\fill}}r}
      \textit{\small#1} & \textit{\small #2} \\
    \end{tabular*}\vspace{-7pt}
}

\newcommand{\resumeProjectHeading}[2]{
    \item
    \begin{tabular*}{0.97\textwidth}{l@{\extracolsep{\fill}}r}
      \small#1 & #2 \\
    \end{tabular*}\vspace{-7pt}
}

\newcommand{\resumeSubItem}[1]{\resumeItem{#1}\vspace{-4pt}}

\renewcommand\labelitemii{$\vcenter{\hbox{\tiny$\bullet$}}$}

\newcommand{\resumeSubHeadingListStart}{\begin{itemize}[leftmargin=0.15in, label={}]}
\newcommand{\resumeSubHeadingListEnd}{\end{itemize}}
\newcommand{\resumeItemListStart}{\begin{itemize}}
\newcommand{\resumeItemListEnd}{\end{itemize}\vspace{-5pt}}

%-------------------------------------------
%%%%%%  RESUME STARTS HERE  %%%%%%%%%%%%%%%%%%%%%%%%%%%%
\begin{document}
%----------HEADING----------
% \begin{tabular*}{\textwidth}{l@{\extracolsep{\fill}}r}
%   \textbf{\href{http://sourabhbajaj.com/}{\Large Sourabh Bajaj}} & Email : \href{mailto:sourabh@sourabhbajaj.com}{sourabh@sourabhbajaj.com}\\
%   \href{http://sourabhbajaj.com/}{http://www.sourabhbajaj.com} & Mobile : +1-123-456-7890 \\
% \end{tabular*}

\begin{center}
    \vspace*{0.01in} \textbf{\Huge \scshape Kyle Spink} \\ \vspace{1pt}
    \small (805) 455-6134 $\big |$ \href{mailto:kspink@uci.edu}{\underline{kspink@uci.edu}} $\big |$ 
    \href{https://www.linkedin.com/in/kyle-spink/}{\underline{linkedin.com/kylespink}} $\big |$
    \href{https://2023mechatronicskylespink.weebly.com/}{\underline{kylespink.com}}
\end{center}

%-----------EDUCATION-----------
% \vspace*{-0.25in}

\section{Education}
  \resumeSubHeadingListStart
    \resumeSubheading
      {University of California, Irvine}{Expected June 2027}
      {Bachelor of Science in Aerospace Engineering}{Irvine, CA}
\vspace{-0.25in}
    \resumeSubheading
      {}{}
      {Bachelor of Science in Applied \& Computational Mathematics}{}   
\vspace{-0.25in}
    \resumeSubheading
      {}{} 
      {\hspace{0.16in} {\scalebox{0.70}{\textbullet \phantom{.}}} \textbf{Relevant Coursework:} Fluid Dynamics \scalebox{1.5}{$\cdot$} Lightweight Structures \scalebox{1.5}{$\cdot$} Partial Differential Equations \scalebox{1.5}{$\cdot$}}{}
\vspace{-0.25in}
    \resumeSubheading
      {}{}
      {\hspace{0.25in} Dynamical Systems (graduate) \scalebox{1.5}{$\cdot$} Numerical ODEs \& Linear Algebra (graduate) \scalebox{1.5}{$\cdot$} Calculus of Variations (graduate)}{}
\vspace{-0.25in}
    \resumeSubheading
      {}{} 
      {\hspace{0.16in} {\scalebox{0.70}{\textbullet \phantom{.}}} \textbf{Organizations:} American Institute of Aeronautics and Astronautics \scalebox{1.5}{$\cdot$} UCI Math Community }{}
\vspace{-0.25in}
    \resumeSubheading
      {}{}
      {\hspace{0.25in} Educational Outreach \scalebox{1.5}{$\cdot$} UCI Undergraduate Math Committee \scalebox{1.5}{$\cdot$} Sigma Eta Pi}{}

  \resumeSubHeadingListEnd

%-----------EXPERIENCE-----------
\section{Experience}
  \resumeSubHeadingListStart
    \resumeSubheading
      {Structural Engineering Intern}{Jun. 2025 -- Present}
      {Northrop Grumman}{Goleta, CA}

    \resumeSubheading
      {Undergraduate Research Assistant}{Mar. 2024 -- Jan. 2025}
      {Aeronautics, Dynamics, \& Control Laboratory}{Irvine, CA}
      \resumeItemListStart
        \resumeItem{Integrated a physics-informed neural network (PINN) with the Principle of Minimum Pressure Gradient (PMPG) to perform computationally inexpensive fluid dynamics simulations (PyTorch)}
        \resumeItem{Coauthor of \textit{\href{https://arc.aiaa.org/doi/abs/10.2514/6.2025-0732}{Predicting Magnus Force with Gauss-Constrained PINNs}} (presented in 2025 AIAA SciTech Forum)}
      \resumeItemListEnd

% -----------Multiple Positions Heading-----------
%    \resumeSubSubheading
%     {Software Engineer I}{Oct 2014 - Sep 2016}
%     \resumeItemListStart
%        \resumeItem{Apache Beam}
%          {Apache Beam is a unified model for defining both batch and streaming data-parallel processing pipelines}
%     \resumeItemListEnd
%    \resumeSubHeadingListEnd
%-------------------------------------------

    \resumeSubheading
      {Engineering Intern}{June 2023 -- Sept. 2023}
